src/input/importOperations.ts

```diff
@@ -27,17 +27,17 @@ const sanitizeInputedAddress = (address: string): string => {
  * @returns string
  *          Imported contents
  */
-const getFileContents = (path: string): string => {
+const getFileContents = (path: string): string | void => {
   if (!fs.existsSync(path)) {
     throw new Error("Imported file " + path + " does not exist");
   }
-
   try {
-    const contents = fs.readFileSync(path, "utf-8");
-    return contents;
-  } catch (err) {
-    console.log(chalk.red("File error"));
-    throw new Error(err);
+    return fs.readFileSync(path, "utf-8");
+  } catch (err: unknown) {
+    if (err instanceof Error) {
+      console.log(chalk.red("File error"));
+      throw new Error(err.message);
+    }
```

Comment:
did you get any cases where `catch` block catched something else than an `Error`?

Reply:
Not to my knowledge. But without this syntax, I do have compilation errors.

Reply:
Updated accordingly.